src/action.cr

```diff
@@ -32,7 +33,10 @@ class Action
   end
 
   def run
-    if perform_shell_expansion?
+    if perform_param_expansion?
+      Output.debug("PARAM EXPANDING")
+      interpreter.exec(@args)
```

Comment:
Could there be subtle differences (other than positional arg expansion) between executing via `Interpreter` and doing a `Process.quote` + `Process.exec`?